backend/public/common/functions.js

```diff
@@ -0,0 +1,15 @@
+function diffDisplay(diff) {
+  console.assert(typeof(diff) === "number", `Diff ${diff} is not a number`);
+
+  // Round the error to 3 digits past the decimal
+  return `${Math.round(diff * 100000) / 1000}%`;
+} // function diffDisplay(diff)
```

Comment:
I found it really hard to keep the file organized since it had local functions inside functions and looking at a `}` alone, I got confused which function actually was closed, so I experimented with this (similar to how we do the namespace closing in the OpenSpace code base) and kinda liked it